app/containers/TextInput/FormTextInput.tsx

```diff
@@ -100,7 +100,7 @@ export const FormTextInput = ({
 	return (
 		<View
 			accessible
-			accessibilityLabel={`${label} - ${required ? i18n.t('Required') : ''}`}
+			accessibilityLabel={accessibilityLabel ?? `${label} - ${required ? i18n.t('Required') : ''}`}
```

Comment:
If it's not required, it's still appending `- ''`.
We should remove that suffix.

Reply:
adjusted.